wagtail/admin/static_src/wagtailadmin/js/core.js

```diff
@@ -57,16 +57,23 @@ function enableDirtyFormCheck(formSelector, options) {
     var $form = $(formSelector);
     var confirmationMessage = options.confirmationMessage || ' ';
     var alwaysDirty = options.alwaysDirty || false;
-    var initialData = $form.serialize();
+    var initialData = null;
```

Comment:
It might be nice to change this from null to an empty string so we can see that it’s going to be a string.

Definitely not a suggestion that has any impact on functionality.

Reply:
To me `null` communicates "not yet defined" better than an empty string (which implies "this is defined and has empty content"), so I'm inclined to keep this as is.